src/docs/build-and-earn/testnet-guides/run-full-node.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Run a Full Node
+---
+
+## ﻿Run a Full Node
+
+Before running a node, make sure that you have followed the guide for [setting up your prerequisites and environment](/docs/build-and-earn/testnet-guides/prerequisites.md).
+
+To run a full node, you need to complete the following steps:
+
+- Initialize the node
+- Configure and start the node
+
+## Initialize the node as a validator
+
+1. Navigate to the directory *CudosBuilders/docker/full-node*
+2. Find the file **full-node.env.example** and create a copy of it
+3. Rename the copied file to **full-node.client.testnet.public01.env**
+4. Open the file, which you renamed, **full-node.client.testnet.public01.env** in any editor
+5. Find both the **"MONIKER"** attribute and the flag **"SHOULD_USE_GLOBAL_PEERS"** and set them as the following (if you are not a validator set the flag to true):
```

Comment:
since it's in a *Initialize the node as a validator* section this part is probably not needed:
"... (if you are not a validator set the flag to true)"